### PR TITLE
refactor(demo): use standalone approach for stackblitz

### PR DESCRIPTION
**Pull request by nsbarsukov**

### CODE REVIEW

File: projects/demo/src/modules/app/stackblitz/project-files/configs.md
Comment (nsbarsukov) [inline projects/demo/src/modules/app/stackblitz/project-files/configs.md:1]: `angular.json` & `tsconfig.json` are copy-pasted from the most recent angular stackblitz starter (angular 18+):

https://github.com/stackblitz/starters/blob/main/angular/tsconfig.json

File: projects/demo/src/modules/app/stackblitz/project-files/src/global_styles.less.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
 ```less
 @import '@taiga-ui/core/styles/taiga-ui-theme.less';
-@import '@taiga-ui/core/styles/taiga-ui-fonts.less';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/stackblitz/project-files/src/global_styles.less.md:3]: This line throws 
```
Error in src/global_styles.less
Error: ENOENT: No such file or directory., '/is.com/css2?family=Manrope:wght@500;800&display=swap'
```

I'll investigate it later in separate PR

File: projects/demo/src/modules/app/stackblitz/stackblitz.service.ts [hunk 2]
```diff
@@ -106,31 +76,15 @@ export class TuiStackblitzService implements TuiCodeEditor {
...
-        additionalModules: Array<[fileName: string, parsedFile: TsFileModuleParser]> = [],
-    ): Promise<Project['files']> {
-        const {stackblitzReadMe} =
-            await AbstractTuiStackblitzResourcesLoader.getReadMeFiles();
-
-        return {
-            [stackblitzPrefix`README.md`]: stackblitzReadMe,
-            [stackblitzPrefix`all-taiga-modules.ts`]:
-                await getAllTaigaUIModulesFile(additionalModules),
+            'src/global_styles.less': globalStyles,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/stackblitz/stackblitz.service.ts:87]: Naming is changed to be similar to the most revent version of angular stackblitz starter

https://github.com/stackblitz/starters/blob/main/angular/src/global_styles.css

File: projects/demo/src/modules/app/stackblitz/project-files/configs.md
Comment (nsbarsukov) [inline projects/demo/src/modules/app/stackblitz/project-files/configs.md:1]: `angular.json` & `tsconfig.json` are copy-pasted from the most recent angular stackblitz starter (angular 18+):

* https://github.com/stackblitz/starters/blob/main/angular/angular.json
* https://github.com/stackblitz/starters/blob/main/angular/tsconfig.json

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/app/stackblitz/project-files/src/global_styles.less.md (hunk 1 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/app/stackblitz/project-files/src/global_styles.less.md:3]: Fixed ✅

File: projects/demo/src/modules/app/stackblitz/project-files/configs.md [hunk 3]
```diff
@@ -3,54 +3,59 @@
 ```json
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/stackblitz/project-files/configs.md:3]: `angular.json` & `tsconfig.json` are copy-pasted from the most recent angular stackblitz starter (angular 18+):

* https://github.com/stackblitz/starters/blob/main/angular/angular.json
* https://github.com/stackblitz/starters/blob/main/angular/tsconfig.json

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**